sklearn/utils/estimator_checks.py

```diff
@@ -597,7 +598,7 @@ def check_fit2d_1sample(name, estimator_orig):
 
 @ignore_warnings
 def check_fit2d_1feature(name, estimator_orig):
-    # check by fitting a 2d array and prediting with a 1d array
+    # check by fitting a 2d array with only one feature
     rnd = np.random.RandomState(0)
     X = 3 * rnd.uniform(size=(10, 1))
     y = X[:, 0].astype(np.int)
```

Comment:
I feel this is a left-over from the deprecation of accepting 1d X. I guess we should ensure that it either works or there is an informative error message. RANSAC and RandomizedLogisticRegression seem weird. SelectKBest could mention ``n_features`` but otherwise seems good?